### DISCUSSION TITLE
在调整时间表后课表会乱

### DISCUSSION THREAD

**Discussion by Makitoid** [ClassIsland/ClassIsland#579] [Q&A] [answered]
由于最近学校铃声系统暂时关闭，我在时间表加上了晚自习的时间。然后所有课表就乱了，从第二节开始课时全部下移一节
其它使用classisland的班也出现了这种情况。
这个问题在我用1.5.0.3时就发现了，现在1.5.2.2也有
bug or操作错误？

**Comment by HelloWRC** [comment]
感觉可能是同步时间表的更改时产生的 bug

**Reply by Makitoid** [reply]
课表不是随机乱序，而是平移。从哪里开始平移不清楚
那我去issue汇报了

**Comment by Makitoid** [comment]
奇怪，这个bug应该很明显啊，为什么没人提出啊

**Reply by HelloWRC** [reply]
有可能是这个bug的触发条件比较苛刻，或者是写这个功能时没有问题，但到后面就有问题了（

**Reply by MKStoler4096** [reply]
其实是有人触发了但是没当成 Bug ，比如我（

**Comment by DryIce-cc** [comment]
@HelloWRC 感觉是我写 #211 那里把部分逻辑提前到 AddTimePoint 里去了，所以就没有重新 ReSortTimeLayout 那一块出现的问题。

@WHMyyds 稍微试下点刷新按钮有没有用。~这个按钮原本是为便于重排分割线添加的，结果后来发现并不起什么作用（~

**Reply by Makitoid** [reply]
抱歉,已经改了，我也不想再逝一次了

**Comment by Makitoid** [comment]
>点刷新试试
在家里逝了一下``1.5.2.2``，没用

**Comment by DryIce-cc** [comment, marked as answer]
@WHMyyds 我在 `1.5.2.2-Himeko-4712e56(develop/management)` 未能复现，可否提供相关档案文件和录屏？
